smartmodule/examples/filter_look_back/transforms.yaml

```diff
@@ -0,0 +1,6 @@
+transforms:
+  - uses: infinyon/fluvio-smartmodule-filter-lookback@0.1.0
+    with:
+      lookback:
```

Comment:
There is no loopback param in the code

Reply:
Not part of this issue but we should have ability to test loopback using SMDK

Reply:
> There is no `loopback` in the code example. It should be removed. The loopback will be controlled by SPU

This parameter is consumed on SmartEngine and SPU levels. It will come in the next PRs.
The loopback is controlled by SPU, that's correct. But it still needs to get information of how many records need to be read and passed to the SM via `look_back` call, and that is where this parameter comes to play.

Reply:
Can't rely on SmartModule on having parameters since it is dependent on author.   This need to be assume be defined by SPU

Reply:
It is supposed to be defined not by the author but by the user of the SmartModule. Some SmartModules need only the last record (github stars, deduplication by monotonic ids, etc), some SmartModules need to read 1000 last records to build some data structure (bloom filters). Later, we may want to introduce asking for records based on time, etc. **This is a concern of the configuration of the SmartModule**. SPU has no this information.

Reply:
This is how it was discussed in RFD https://github.com/infinyon/infinyon-rfd/pull/23 and already implemented

Reply:
I updated example